packages/hdwallet-portis/src/adapter.ts

```diff
@@ -5,40 +5,35 @@ import {
 
 import { PortisHDWallet } from './portis'
 import Portis from "@portis/web3";
-import Web3 from 'web3'
-import { getEthAddress } from './utils'
 
 type PortisWallet = any
 
 export class PortisAdapter {
     keyring: Keyring
     portis: any
   
-    private constructor (keyring: Keyring, args: { portis?: PortisWallet, portisAppId?: string }) {
+    private constructor (keyring: Keyring) {
```

Comment:
I think we should still pass the `AppId` in via `useKeyring()`, as that will be a more natural place to provide it from WalletContext than to have to pass it to `pairDevice()`.

Reply:
done